dbt/models/qc/schema.yml

```diff
@@ -231,7 +231,9 @@ models:
       export_template: &qc_report_ic_reference_export_template
         start_row: 3
         add_table: false
-      export_format:
+        sheet_name: Query Results
+      export_format: &qc_report_ic_reference_export_format
+        format_blanks_as_empty_string: true
```

Comment:
The sheet name feels to me like a template configuration option, while the missingness value feels like a format configuration option. The boundary here is somewhat arbitrary, however, so I'm open to different approaches.